src/Types.jl

```diff
@@ -1240,13 +1244,13 @@ function find_registered!(ctx::Context,
     for registry in collect_registries()
         reg_abspath = abspath(registry.path)
         data = read_registry(joinpath(registry.path, "Registry.toml"))
-        for (_uuid, pkgdata) in data["packages"]
-              uuid = UUID(_uuid)
-              name = pkgdata["name"]
-              path = joinpath(reg_abspath, pkgdata["path"])
-              push!(get!(ctx.env.uuids, name, UUID[]), uuid)
-              push!(get!(ctx.env.paths, uuid, String[]), path)
-              push!(get!(ctx.env.names, uuid, String[]), name)
+        for (_uuid, pkgdata) in data["packages"]   # what types do these have?
```

Comment:
Anyone know what types `_uuid` and `pkgdata` are?

Reply:
`_uuid` should be `String` and `pkgdata` a `Dict{String, Any}` I think.